src/libstd/f32.rs

```diff
@@ -912,6 +912,30 @@ impl f32 {
         }
         x
     }
+
+    /// Computes `self + step(upper - pol)` the linear interpolation between
```

Comment:
Not sure what `step` refers to but this description doesn't look like it matches the implementation. In particular `pol` is [dimensionless](https://en.wikipedia.org/wiki/Dimensionless_quantity) and `upper` is not so `upper - pol` is not an expression that would make sense.